openquakeplatform_building_class/templates/building-class/user-settings.html

```diff
@@ -44,6 +44,8 @@
           <h1>{% trans "Building Classification Survey" %}</h1>
           <h2>{% trans "User settings" %}</h2>
           <br />
+          The 'Building classification tool' aims to create a global inventory of
+building classes in the world.<br><br>
```

Comment:
Remove "in the world" - this is redundant since we already said the inventory is global